test/exchanges/oasis/Eth2DaiDirect.spec.js

```diff
@@ -0,0 +1,223 @@
+import { buildTestService } from '../../helpers/serviceBuilders';
```

Comment:
what could be done to make this test run faster? right now, on my machine, it increases the runtime of the entire test suite by almost 50%

Reply:
By reworking the helper functions we use in both exchange services, I was able to cut this down to about 25 seconds. That's still longer than most, but I think it's probably about as little time as possible for so many transactions.